docs/reference/QueryCache.md

```diff
@@ -71,7 +71,7 @@ const query = queryCache.find(queryKey)
 > Note: This is not typically needed for most applications, but can come in handy when needing more information about a query in rare scenarios
 
 ```tsx
-const queries = queryCache.findAll(queryKey)
+const queries = queryCache.findAll([queryKey])
```

Comment:
i would leave these two as-is. the options then state that queryKey needs to be of type QueryKey:

```
`queryKey?: QueryKey`: [Query Keys](../guides/query-keys)
```

and `QuerKey` is already an array!

Reply:
That's a good point, missed that, sorry!